src/pages/LikedPage/LikedPage.jsx

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { LikedCard } from "../../components";
+import * as styles from "./LikedPage.module.css";
+export function LikedPage() {
+  const [likedVideos, setLikedVideos] = useState([]);
+  const getLikedVideos = async () => {
+    try {
+      const encodedToken = localStorage.getItem("Manaplay.encodedToken");
+      const response = await axios.get("/api/user/likes", {
+        headers: { authorization: encodedToken },
+      });
+
+      setLikedVideos(response.data.likes);
+    } catch (error) {
```

Comment:
you can check for status code as you check in previous request

Reply:
yes i will update it